packages/phone_log/example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Jun 23 08:50:38 CEST 2017
```

Comment:
You only need this file. Revert everything else in this directorly. The updated .gitignore should handle all this.